docs/sources/panels/transformations/fields-to-labels.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Fields to labels"
```

Comment:
Do we need a new topic here? Let's add this information in the "Transformation types and options" topic, as a section after https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#labels-to-fields.